contracts/CTKNCrowdsale.sol

```diff
@@ -44,4 +75,71 @@ contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale
         dollarRate = _dollarRate;
         DollarRateSet(dollarRate);
     }
+
+    /**
+     *  Get the amount the given address has overpaid.
+     *  @param addr The address to check for an overpayment balance
+     *  @return the number of `wei` the address overpaid when buying tokens.
+     */
+    function overpaymentBalance(address addr) external view returns (uint256) {
+        return overpaymentVault.deposited(addr);
```

Comment:
I see occasional checks in the zeppelin codebase for != address(0) to see if the address has been created.  Is that necessary here?

Reply:
No because `address(0)` will not have deposited, so will return a balance of `0`